ansible-scylla-node/tasks/common.yml

```diff
@@ -285,6 +285,11 @@
   set_fact:
     broadcast_address: "{{ scylla_broadcast_address }}"
 
+- name: Set start_scylla_service to false if there is an ongoing replacement
+  set_fact:
+    _start_scylla_service: |
+      {% if (start_scylla_service and replacing_node == false) %}True{% else %}False{% endif %}
```

Comment:
That's a little clumsy expression. 
Why not simply:

```
- name: Set start_scylla_service to false if there is an ongoing replacement
  set_fact:
    _start_scylla_service: '{{ start_scylla_service == true and replacing_node == false }}'
```
??

Reply:
Fixed